cluster-controller/src/main/java/io/strimzi/controller/cluster/ClusterControllerConfig.java

```diff
@@ -69,13 +74,19 @@ public static ClusterControllerConfig fromMap(Map<String, String> map) {
         if (stringLabels == null || stringLabels.isEmpty()) {
             throw new IllegalArgumentException(ClusterControllerConfig.STRIMZI_CONFIGMAP_LABELS + " cannot be null");
         }
-        Long reconciliationInterval = DEFAULT_FULL_RECONCILIATION_INTERVAL;
 
+        Long reconciliationInterval = DEFAULT_FULL_RECONCILIATION_INTERVAL;
         String reconciliationIntervalEnvVar = map.get(ClusterControllerConfig.STRIMZI_FULL_RECONCILIATION_INTERVAL);
         if (reconciliationIntervalEnvVar != null) {
             reconciliationInterval = Long.valueOf(reconciliationIntervalEnvVar);
         }
 
+        Long operationTimeout = DEFAULT_OPERATION_TIMEOUT;
```

Comment:
Is it really necessary for this to be a `Long`? You could use a primitive and `Long.parseLong()` instead of `Long.valueOf()`.